### PR TITLE
Fix nuget package dependencies 

### PR DESCRIPTION
**Pull request by pakrym**
@davidfowl

### CODE REVIEW

File: tools/common.props [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<Project>
+  <PropertyGroup>
+    <VersionSuffix>$([System.DateTime]::Now.ToString(eyyMMdd-1))</VersionSuffix>
```
Comment (davidfowl) [inline tools/common.props]: `eyyMMdd-1` doesnt' have to be in quotes?

**Comment by davidfowl** [comment]
@dotnet-bot test this please

**Comment by davidfowl** [comment]
@dotnet-bot test Windows Release please

**Comment by pakrym** [comment]
@dotnet-bot test Windows Release please

File: tools/common.props [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    <Authors>Microsoft Corporation</Authors>
+    <PackageReleaseNotes>Pre-release package, for testing only</PackageReleaseNotes>
+    <PackageIconUrl>http://go.microsoft.com/fwlink/?LinkID=288859</PackageIconUrl>
+    <PackageProjectUrl>https://github.com/dotnet/corefxlab</PackageProjectUrl>
+    <PackageLicenseUrl>http://go.microsoft.com/fwlink/?LinkId=329770</PackageLicenseUrl>
+    <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+    <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
+    <AssemblyOriginatorKeyFile>../../tools/Key.snk</AssemblyOriginatorKeyFile>
+  </PropertyGroup>
+</Project>
```
Comment (ahsonkhan) [inline tools/common.props]: How is this .props file used during the build/packaging step?

Comment (davidfowl) [inline tools/common.props]: It's included in all of the projects and it sets the version for each project.

Comment (ahsonkhan) [inline tools/common.props]: Got it. Thanks :)

File: tools/common.props (hunk 1 above)
Comment (ahsonkhan) [inline tools/common.props]: Does the "e" specifier mean anything or is it just string?
I couldn't find it anywhere here:
https://msdn.microsoft.com/en-us/library/8kb3ddd4.aspx

```C#
Console.WriteLine(System.DateTime.Now.ToString("eyyMMdd-1"));
```
**Output:** e170206-1

Comment (pakrym) [inline tools/common.props]: just a string, see https://dotnet.myget.org/feed/dotnet-corefxlab/package/nuget/System.IO.Pipelines/0.1.0-e170206-2

Comment (ahsonkhan) [inline tools/common.props]: Thanks.

So, for example: "0.1.0-e170206-2"
Where does the hyphen come from between the version prefix and version suffix? Or does it automatically concatenate with hyphen in between?
```C#
<VersionPrefix>0.1.0</VersionPrefix> 
<VersionSuffix>$([System.DateTime]::Now.ToString(eyyMMdd-1))</VersionSuffix> 
```

Comment (pakrym) [inline tools/common.props]: Yes, NuGet pack concats with hyphen.

**Comment by pakrym** [comment]
@dotnet-bot test this please

**Pull request merged**